k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -3397,14 +3397,17 @@ private void moveOrDeleteSentMessage(Account account, LocalStore localStore,
 
 
             localSentFolder.appendMessages(Collections.singletonList(message));
-            //localFolder.moveMessages(Collections.singletonList(message), localSentFolder);
-            PendingCommand command = PendingAppend.create(localSentFolder.getName(), message.getUid());
-            queuePendingCommand(account, command);
-            processPendingCommands(account);
-
-            Timber.i("Moved sent message to folder '%s' (%d)",
-                    account.getSentFolderName(), localSentFolder.getId());
-
+            if (isLocalMessageAlreadyOnServer(account, encryptedMessage.getMessageId(), account.getSentFolderName())) {
+                Timber.d("EFA-656 Sent message is already in remote folder, nothing to do");
+            } else {
+                Timber.d("EFA-656 Sent message not found on remote folder, appending it");
+                //localFolder.moveMessages(Collections.singletonList(message), localSentFolder);
```

Comment:
```suggestion
```

You removed the function parameter, so no real need to keep this line around imho. Also, it is already appended to the localSentFolder above.

Reply:
Removed, thanks.